File: backend/FastFashion/Models/FashionItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace FastFashion.Models
{
    public class FashionItem
    {

        public int Id { get; set; }

        [Required]
        public string Type { get; set; }

        public string Detail { get; set; }

        public string Style { get; set; }

        public string Date { get; set; }

        public string pictureUri { get; set; }

        public int UserId { get; set; }


    }
}

File: frontend/app/src/main/java/com/example/fastfashion/model/FashionItem.kt
package com.example.fastfashion.model

import com.google.gson.annotations.SerializedName

data class FashionItem (
    @SerializedName("id")
    val id: Int,
    @SerializedName("type")
    val type: String,
    @SerializedName("detail")
    val detail: String,
    @SerializedName("style")
    val style: String,
    @SerializedName("date")
    val date: String,
    @SerializedName("pictureUri")
    val uri: String
)